templates/paragraphs/paragraph--contact-card.html.twig

```diff
@@ -0,0 +1,52 @@
+{%
+  set classes = [
+    'contact-card',
+    view_mode ? 'contact-card---' ~ view_mode|clean_class,
+    content.field_contact_image|render ? 'contact-card--with-image'
+  ]
+%}
+
+
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% if content.field_contact_image|render %}
+      <div class="contact-card--image-container">
+        {{ content.field_contact_image }}
+      </div>
+    {% else %}
+      <div class="contact-card--color-line"></div>
```

Comment:
What if you'd move the contact-card--image-container outside of the `if` and have no extra element just for the line?